app/controllers/users.js

```diff
@@ -39,21 +39,22 @@ exports.signup = function(req, res) {
  */
 exports.signout = function(req, res) {
   req.logout();
-  res.redirect('/');
+  res.redirect('/#!/removeToken/' + 1);
```

Comment:
There is no need having to explicitly add a 1 here. Just check that a token exists in localstorage and remove it.